classes/SectionView/SectionView.php

```diff
@@ -160,6 +160,12 @@ protected function display_section_or_speech($args = array()) {
         }
 
         // Do all this unless the glossary is turned off in the URL
+        if ($SEARCHENGINE) {
```

Comment:
The comment line above is referring to the line after this added block (so this block needs moving up a line or two, or put back where it was if the other commit now fixes it a different way).